test/unit/artefact/artefact_test.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../../test_helper'
+
+class Artefact::ArtefactTest < ActiveSupport::TestCase
+
+  context "taggable?" do
+
+    should "return true for artifacts not owned by publisher, smartanswers or testapp" do
+      Settings.expects(:untaggable_apps).returns(%w{ publisher smartanswers testapp }).at_least(1)
```

Comment:
These could be extracted to a higher context and shared across the various `should` here.